Rakefile

```
@@ -0,0 +1,5 @@
+
+PODSPEC_FILE = 'Stencil.podspec.json'
+CHANGELOG_FILE = 'CHANGELOG.md'
+
```

Add this bit to ensure we're using bundler (copied from SwiftGen `Rakefile`):
```suggestion

if ENV['BUNDLE_GEMFILE'].nil?
  puts "\u{274C} Please use bundle exec"
  exit 1
end

```